README.md

```diff
@@ -5,14 +5,37 @@ Firefox Accounts DB Server
 
 ## Usage
 
+If you would like to run a FxA Auth DB Server, then you probably want to look at the
```

Comment:
"run _an_ FxA Auth"? or maybe "run a _Firefox Accounts_ Auth"?
I dunno, English is hard.